dev/com.ibm.ws.microprofile.config.1.3/src/com/ibm/ws/microprofile/config13/converters/Config13ImplicitConverter.java

```diff
@@ -43,138 +42,60 @@
-
-        return method;
-    }
-
-    @FFDCIgnore(NoSuchMethodException.class)
-    @Trivial
-    private static Method getValueOfMethod(Class<?> reflectionClass) {
-        Method method = null;
-        try {
-            method = reflectionClass.getMethod("valueOf", String.class);
-            if ((method.getModifiers() & Modifier.STATIC) == 0) {
-                method = null;
-            } else if (!reflectionClass.equals(method.getReturnType())) {
-                method = null;
-            }
-        } catch (NoSuchMethodException e) {
-            //No FFDC
-        }
-
-        return method;
-    }
-
-    @FFDCIgnore(NoSuchMethodException.class)
-    @Trivial
-    private static Method getParse(Class<?> reflectionClass) {
-        Method method = null;
-        try {
-            method = reflectionClass.getMethod("parse", CharSequence.class);
-            if ((method.getModifiers() & Modifier.STATIC) == 0) {
-                method = null;
-            } else if (!reflectionClass.equals(method.getReturnType())) {
-                method = null;
-            }
-        } catch (NoSuchMethodException e) {
-            //No FFDC
-        }
-        return method;
+        return implicitFunction;
     }
 
     /** {@inheritDoc} */
     @Override
     public Object convert(String value) {
-        Object converted = null;
-        if (value != null) { //if the value is null then we always return null
-            try {
-
-                if (this.ofMethod != null) {
-                    converted = this.ofMethod.invoke(null, value);
-                } else if (this.valueOfMethod != null) {
-                    converted = this.valueOfMethod.invoke(null, value);
-                } else if (this.ctor != null) {
-                    converted = this.ctor.newInstance(value);
-                } else if (this.parseMethod != null) {
-                    converted = this.parseMethod.invoke(null, value);
-                }
-            } catch (InvocationTargetException e) {
-                Throwable cause = e.getCause();
-                if (cause instanceof IllegalArgumentException) {
-                    throw (IllegalArgumentException) cause;
-                } else {
-                    throw new IllegalArgumentException(cause);
-                }
-            } catch (IllegalAccessException | InstantiationException e) {
-                throw new IllegalArgumentException(e);
+        try {
+            return super.convert(value);
+        } catch (ConversionException e) { //The Config 1.3 spec clarified that the convert method should throw IllegalArgumentException
+                                          //if the value cannot be converted to the specified type
+            Throwable cause = e.getCause();
+            if (cause instanceof IllegalArgumentException) {
+                throw (IllegalArgumentException) cause;
+            } else {
+                throw new IllegalArgumentException(cause);
             }
         }
```

Comment:
Should this method have `@FFDCIgnore`? At the moment we would get an FFDC from the `MethodFunction` and then another one here. We shouldn't need both, right?

Reply:
Should probably ignore this one ... FFDCs should occur at the point of first failure; this is just re-wrapping the exception, it doesn't add anything to FFDC here.

Reply:
done